app/controllers/api/stories_controller.rb

```diff
@@ -68,7 +68,7 @@ def included(relation)
   end
 
   def scoped(relation)
-    relation.published.visible
+    relation.published.network_visible.series_visible
```

Comment:
sure. Previously the `visible` scope was really doing 2 things: it filtered out pieces that were `network_only`, or were in a series where the pieces were `subscriber_only`. I made the two scopes for `network_visible` and `series_visible` from the single `visible` scope, because there are conditions when I want to use one or the other but not both.